qiskit/qpy/binary_io/circuits.py

```diff
@@ -1419,7 +1421,20 @@ def write_circuit(
         version (int): The QPY format version to use for serializing this circuit
         annotation_factories (dict): a mapping of namespaces to zero-argument factory functions that
             produce instances of :class:`.annotation.QPYSerializer`.
+        use_rust (bool): whether to use the rust based serialization engine. On by default.
```

Comment:
```suggestion
        use_rust (bool): whether to use the rust based serialization engine. Off by default.
```

The parameter's default value is `False`